components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2267,24 +2267,42 @@ def _annotations(request, objtype, objid, conn=None, **kwargs):
     query += """
```

Comment:
Thanks @emilroz, but it seems to be loading link owners already.
In the json at webgateway/table/Plate.wells/2061/query/?query=Well-2061 I see 
``` "addedBy": "Will Moore" ```
even when logged in as another user whose default group is different from this one.